.github/workflows/ci.yml

```diff
@@ -764,31 +764,36 @@ jobs:
       matrix:
         target:
           - platform: android
-            architecture: arm64-v8a
+            arch: arm_64
             abi: arm64-v8a
             docker_image: "gcr.io/iree-oss/android@sha256:e5f45760ab2dc8a1d72dc6e9d90056719bedac5678ee90e6d9f9b958172b4834"
+            build_script: "./build_tools/cmake/build_android.sh"
             # No test_script
-          - platform: riscv
-            architecture: rv64
+          - platform: linux
+            arch: riscv_64
             abi: lp64d
             docker_image: "gcr.io/iree-oss/riscv@sha256:5034a40068f14d91d5caf07d615f323561416caee8afda32ae622746a4dd8b04"
+            build_script: "./build_tools/cmake/build_riscv.sh"
             test_script: "./build_tools/cmake/test_riscv.sh"
-          - platform: riscv
-            architecture: rv32-linux
+          - platform: linux
+            arch: riscv_32
             abi: ilp32d
             docker_image: "gcr.io/iree-oss/riscv@sha256:5034a40068f14d91d5caf07d615f323561416caee8afda32ae622746a4dd8b04"
+            build_script: "./build_tools/cmake/build_riscv.sh"
             test_script: "./build_tools/cmake/test_riscv.sh"
-          - platform: riscv
-            architecture: rv32-baremetal
+          - platform: baremetal
```

Comment:
I think this is referred to as "generic" in target_platform.h. Not sure if we should call it that here as well

Reply:
Noticed that in [docs/website/docs/deployment-configurations/bare-metal.md](https://iree-org.github.io/iree/deployment-configurations/bare-metal/#define-iree-macros) we also refer "PLATFORM_GENERIC" for bare-metal. I guess it makes sense to also use `generic` here (a little less intuitive).